### DISCUSSION TITLE
This dependency was not found

### DISCUSSION THREAD

**Discussion by rmshmaudlin** [alexcaza/export-to-csv#90] [Q&A]
I'm getting This dependency was not found error even after i installed the package.

**Comment by alexcaza** [comment]
Could you share more details about your project, please? If you can, post your `package.json`, `tsconfig.json` (if applicable), and what context you're using the library in and details about the environment (browser, node, deno, bun, etc)?

I haven't experienced this error before, so it's hard for me to help without more info.
